src/extension.ts

```diff
@@ -139,6 +149,59 @@ export function activate(context: vscode.ExtensionContext): void {
     void vscode.commands.executeCommand('setContext', 'titleCommandsRegistered', true);
 }
 
+export async function attachShare(
+    context: vscode.ExtensionContext, attachUrl?: string, config?: Partial<IUserConfig>, useRetry?: boolean): Promise<void> {
+    if (!telemetryReporter) {
+        telemetryReporter = createTelemetryReporter(context);
+    }
+
+    const telemetryProps = { viaConfig: `${!!config}`, withTargetUrl: `${!!attachUrl}` };
+    telemetryReporter.sendTelemetryEvent('command/attach', telemetryProps);
+
+    const { timeout } = getRemoteEndpointSettings(config);
+
+    // Get the attach target and keep trying until reaching timeout
+    const startTime = Date.now();
+    do {
+        // Attempt to attach to active CDP target
+        const session = vscode.debug.activeDebugSession;
+        if (!session) {
+            vscode.window.showErrorMessage('No active debug session');
+            continue;
+        }
+
+        let targetWebsocketUrl;
+        try {
+            const addr: any = await vscode.commands.executeCommand(
+            'extension.js-debug.requestCDPProxy',
+            session.id,
+            );
+            const formed = `ws://${addr.host}:${addr.port}`;
+            vscode.window.showInformationMessage(`Address copied to your clipboard (${formed})`);
```

Comment:
Does the address get written in the keyboard as a part of `extension.js-debug.requestCDPProxy`?  It isn't a big deal, but not a fan of buttons unexpectedly overwriting a user's clipboard.